errors/error.go

```diff
@@ -206,15 +242,9 @@ func (e *Error) isEmpty() bool {
 	return e.FileRange == hcl.Range{} && e.Kind == "" && e.Description == "" && e.Stack == nil
 }
 
-func (e *Error) error(verbose bool) string {
+func (e *Error) error(members []interface{}, verbose bool) string {
```

Comment:
If we go with defaultErrorFields, could also be defaultErrorMembers:

```suggestion
func (e *Error) error(fields[]interface{}, verbose bool) string {
```